Source/Initialization/WarpXInitData.cpp

```diff
@@ -185,6 +185,56 @@ namespace
         // TODO: check memory per MPI rank, especially if GPUs are underutilized
         // TODO: CPU tiling hints with OpenMP
     }
+
+    /**
+     * \brief Checks for known numerical issues involving different electromagnetic solvers
+     */
+    void CheckKnownEMSolverIssues (
+        const ElectromagneticSolverAlgo em_solver_algo,
+        const CurrentDepositionAlgo current_dep_algo,
```

Comment:
Maybe I would call this `current_deposition_algo`, instead of `current_dep_algo`, just for extra clarity, but it's just a personal preference.

Reply:
I agree with you !  I will do the change